docs/create-a-ui-component.md

```diff
@@ -0,0 +1,280 @@
+in your component file.
+
+```jsx
+// src/web/theme/ui/molecules/MediaCard/MediaCard.js
+import "./MediaCard.scss";
+
+// ... with the rest of your code here
+```
+
+```scss
+// src/web/theme/ui/molecules/MediaCard/MediaCard.scss
+.media-card {
+  display: flex;
+}
+.media-card__media {
+  width: 30%;
+  max-width: 10em;
+}
+```
+
+Please note here, that we are using [Sass](https://sass-lang.com/) (hence the
+`.scss` extension). We believe that it is easier for developers new to the React
+Ecosystem to remain with this well-known CSS preprocessor.
+
+Additionnally, in each Sass files, we will be able to use a list of variables
+that will allow you to keep our theme consitent accross the whole theme. These
+are available at `src/web/theme/globals.scss`.
+
+As a side note, we also use [BEM convention](http://getbem.com/naming/) for our
+CSS code base. It makes it easy to avoid naming conflicts by adding a tiny bit
+of code ceremony. However, for you custom components, feel free to code however
+you like. There is no obligation here.
+
+#### Document usage of our component
+
+If our component can have different usages, we should also add new stories along
+the default one.
+
+```jsx
+// src/web/theme/ui/molecules/MediaCard/MediaCard.story.js
+import MediaCard from "./MediaCard.js";
+import { storiesOf } from "@storybook/react";
+import Icon from "theme/ui/atoms/Icon";
+import Heading from "theme/ui/atoms/Heading";
+import Description from "theme/ui/atoms/Description";
+
+storiesOf("molecules.MediaCard", module)
+  .add("default", () => {
+    return (
+      <MediaCard media={<Icon icon="truck" />}>
+        <Heading>Shipping within 48h</Heading>
+      </MediaCard>
+    );
+  })
+  .add("with a lot of content", () => {
+    return (
+      <MediaCard media={<Icon icon="truck" />}>
+        <Heading>Shipping within 48h</Heading>
+        <Description>
+          We are using many delivery services to let you choose what is best for
+          you!
+        </Description>
+      </MediaCard>
+    );
+  })
+  .add("with an image", () => {
+    return (
+      <MediaCard
+        media={
+          <Image src="http://via.placeholder.com/350x150" alt="Placeholder" />
+        }
+      >
+        <Heading>Shipping within 48h</Heading>
+      </MediaCard>
+    );
+  });
+```
+
+It has many major benefits such as:
+
+* easily document edge cases
+* provide a test suite thanks to snapshot testing
+  ([Storyshots](https://github.com/storybooks/storybook/tree/master/addons/storyshots))
+* create a common discussion base for designers, product managers, marketers,
+  etc.
+
+[Learn more about Storybook.](https://storybook.js.org/)
+
+### Step 4: Use the component
+
+Once we are satisfied with our component, we can use it anywhere. In this case,
+the MediaCard was to be used in the Reinsurance Banner. Thus, this module's
+component would look like this:
+
+```jsx
+import React from "react";
+// Imports can be absolute from `theme`
```

Comment:
We should add « in Front-Commerce (not available in FCLite) » or a note about it

Reply:
We should just add a simple Webpack alias for that.